src/test/kotlin/sudoklin/solver/NewSudokuSolverTest.kt

```diff
@@ -133,15 +132,15 @@ class NewSudokuSolverTest : Spek({
                     assertTrue(solvedSudoku.isValid())
                 }
 
-/*                it("should be able to solve an medium puzzle") {
+                it("should be able to solve an medium puzzle") {
```

Comment:
this test is failing.
Do we need more advanced algorithms before we can solve this?